### DISCUSSION TITLE
Support for coroutines in rest client

### DISCUSSION THREAD

**Discussion by hennihaus** [quarkusio/quarkus#25629] [Q&A]
Do you know if there is REACTIVE support for clients with the suspend keyword? Is this non-blocking or blocking code in quarkus?

![Bildschirmfoto 2022-05-17 um 21 01 52](https://user-images.githubusercontent.com/47771386/168890315-e58a233a-2652-492e-82fb-d03c9a61ed67.png)

[a comment by an automated account is not shown]

**Reply by mschorsch** [reply]
Resteasy Reactive Rest Client supports kotlin coroutines (https://quarkus.io/guides/kotlin#extensions). You have to add quarkus-kotlin.

Suspend functions are always nonblocking.

**Reply by kchvilyov** [reply]
Thank you! You saved my life! :)
